crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -226,12 +226,12 @@ pub struct GhostParentWrapperDyn<
 }
 
 impl<
-        Context,
+        Context: 'static,
         RequestToParent,
         RequestToParentResponse,
         RequestToChild,
-        RequestToChildResponse,
-        Error,
+        RequestToChildResponse: 'static,
+        Error: 'static,
```

Comment:
These two should be fine... we should probably add `'static' to all the Request* types, because we don't want references in those or the error type.

Reply:
@neonphog why don't we want references in request types? Because we need to serialize them,  or share among threads?

Reply:
Exactly, If we do end up wanting to upgrade this to a "managed" actor framework where we don't have to manually call process on each child, we'd need these messages to be passable without references.